retrospective-update/functions.py

```
@@ -0,0 +1,1036 @@
+    """
+    return ds[[qout]].reset_coords(drop=True)
+
+
+def concatenate_outputs(outputs_dir: str,
+                        hourly_zarr_path: str,
+                        daily_zarr_path: str,
+                        CL: CloudLog) -> None:
+    # Build the week dataset
+    qouts = natsort.natsorted(glob.glob(os.path.join(outputs_dir, '*', 'Qout*.nc')))
+    if not qouts:
+        CL.ping('FAIL', f"No-Qout-files-found")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)